### PR TITLE
OCPCLOUD-2787: Updates migration controller cmd

### PR DESCRIPTION
**Pull request by theobarberbany**
Fixes broken import of capa v1beta2, removes capav1beta1.

### CODE REVIEW

File: cmd/machine-api-migration/main.go [hunk 1]
```diff
@@ -29,7 +29,8 @@ import (
 	"github.com/openshift/cluster-capi-operator/pkg/controllers/machinesetsync"
 	"github.com/openshift/cluster-capi-operator/pkg/controllers/machinesync"
 	"github.com/openshift/cluster-capi-operator/pkg/util"
-	capav1beta1 "sigs.k8s.io/cluster-api-provider-aws/v2/api/v1beta2"
+	capav1beta1 "sigs.k8s.io/cluster-api-provider-aws/v2/api/v1beta1"
```
Comment (damdo) [inline cmd/machine-api-migration/main.go:32]: Do we need the v1beta1 anywhere?

Comment (theobarberbany) [reply cmd/machine-api-migration/main.go:32]: Hmm we currently use v1beta1 in the machine-sync controller `return &awscapiv1beta1.AWSMachine{}`, but if we can get the same from v1beta2, I don't see why we couldn't remove it..

Comment (theobarberbany) [reply cmd/machine-api-migration/main.go]: Ah that makes sense, I have updated :) 

> v1beta2, as that's what we work with everywhere,

Must have missed this on the machineset sync reviews before, oops

**Review by damdo — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**